### ISSUE TITLE
Name the binary "git-town"

### ISSUE THREAD

**Issue by kevgo** [Originate/git-town#820]
If the binary is named `git-town`, the user can run:
* `git town sync` 
* `git-town sync`

In addition, the installer can provide an alias named `gt` for ease of use.

**Comment by charlierudolph** [comment]
I'd suggest we pick just one and let the user alias if they would like. Setting up aliases appears to be very simple in bash and fish.

http://tldp.org/LDP/abs/html/aliases.html
http://fishshell.com/docs/current/commands.html#alias

I think I'd favor using `git-town` over `gt` as it uses the full name and with the go release I don't see any changes to my current usage as we can setup the aliases in the same way

The current aliases would then just change from 
```
git config --global alias.sync town-sync
```
to
```
git config --global alias.sync "town sync"
```

**Comment by charlierudolph** [comment]
Thoughts on holding off on this until all commands are rewritten in go? Otherwise this would cause an ambiguity in the tests as we would have multiple git-town binaries

**Comment by kevgo** [comment]
Yes, I just wanted to capture this idea. We can change this once we have converted everything (and broken up the old `git town` command).

**Comment by charlierudolph** [comment]
Fixed in #885
